examples/mesh_test.c

```
@@ -1,417 +1,155 @@
+    hello.responder_id = 0x0002;
+    memcpy(hello.public_key, eph_pub, PUBLIC_KEY_LENGTH);
+
+    /* Store responder identity keys for simulation */
+    uint8_t resp_pub[PUBLIC_KEY_LENGTH], resp_priv[PRIVATE_KEY_LENGTH];
+    crypto_box_keypair(resp_pub, resp_priv);
+    security_store_identity_keys(resp_pub, resp_priv);
+
+    /* Switch to responder perspective temporarily by storing its keys */
+    /* For this test we simulate by calling process_handshake directly */
+
+    uint8_t orig_pub[PUBLIC_KEY_LENGTH], orig_priv[PRIVATE_KEY_LENGTH];
```

Clang (14.0.6)

[warning] variable 'orig_pub' is not initialized

(cppcoreguidelines-init-variables)